client/post-editor/editor-drawer/term-selector.jsx

```diff
@@ -0,0 +1,80 @@
+/**
```

Comment:
Should we move this to its own directory, `editor-term-selector` ? My thinking with files within `editor-drawer` is that they should be limited to the logic necessary to render the accordion itself, but using components from their own directories.